x-pack/plugin/ml/qa/native-multi-node-tests/src/test/java/org/elasticsearch/xpack/ml/integration/ClassificationIT.java

```diff
@@ -248,6 +250,27 @@ public void testDependentVariableCardinalityTooHighError() {
         assertThat(e.getMessage(), equalTo("Field [keyword-field] must have at most [2] distinct values but there were at least [3]"));
     }
 
+    public void testDependentVariableCardinalityTooHighErrorButWithQuery() throws Exception {
+        initialize("cardinality_too_high");
```

Comment:
Please make the names assigned to jobs unique (currently this one and the one in line 236 are the same).